### PR TITLE
Alerts component for storage overview tab

### PR DESCRIPTION
**Pull request by cloudbehl**

### CODE REVIEW

File: src/components/Dashboard/Alert/fixtures/AlertsBody.fixture.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import { AlertsBody } from '../AlertsBody';
-import { criticalAlert, unknownTypeAlert, warningAlert } from './AlertItem.fixture';
+import { cephDiskInaccessibleAlert, watchDogAlert, cephDataRecoveryAlert } from './AlertItem.fixture';
 import { AlertItem } from '../AlertItem';
 
 export default {
   component: AlertsBody,
   props: {
     children: [
-      <AlertItem key="1" alert={criticalAlert} />,
```
Comment (rawagner) [inline src/components/Dashboard/Alert/fixtures/AlertsBody.fixture.js]: why changing these generic alerts to specific ones ?

File: src/components/Dashboard/Alert/fixtures/AlertItem.fixture.js [hunk 2]
```diff
@@ -40,6 +40,30 @@ export const unknownTypeAlert = {
   },
 };
 
+export const cephDiskInaccessibleAlert = {
```
Comment (rawagner) [inline src/components/Dashboard/Alert/fixtures/AlertItem.fixture.js]: I suggest to move this ceph-specific alerts to StorageOverview `Alerts.fixture.js`

Comment (cloudbehl) [reply src/components/Dashboard/Alert/fixtures/AlertItem.fixture.js]: ack!

File: src/components/Dashboard/Alert/fixtures/AlertsBody.fixture.js (hunk 1 above)
Comment (cloudbehl) [reply src/components/Dashboard/Alert/fixtures/AlertsBody.fixture.js]: Sorry, I missed it. I thought I am changing in storage tab. Thanks

made the required changes.

File: src/components/StorageOverview/Alerts/utils.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { get } from 'lodash';
+
+export const filterAlerts = alerts => alerts.filter(alert => get(alert, 'annotations.storage_type') === 'ceph');
```
Comment (suomiy) [inline src/components/StorageOverview/Alerts/utils.js]: ` get(alert, 'annotations.storage_type') ` should go to `selectors/prometheus/alerts.js`

Comment (cloudbehl) [reply src/components/StorageOverview/Alerts/utils.js]: @suomiy I think it's specific to ceph cluster and will not be used for any other filtering. can you tell me why you think so?

Comment (suomiy) [reply src/components/StorageOverview/Alerts/utils.js]: it is better to split the selectors from every place they are used in. So maybe to `selectors/prometheus/alerts/storage.js` ? and move also alerts.js to that folder ?

Comment (cloudbehl) [reply src/components/StorageOverview/Alerts/utils.js]: ack! Got it. Thanks

File: src/components/StorageOverview/StorageOverview.js [hunk 4]
```diff
@@ -12,12 +12,20 @@ import OCSHealthConnected from './OCSHealth/Health';
...
   <GridItem lg={6} md={12} sm={12}>
     <Grid>
       <GridItem span={12}>
-        <OCSHealthConnected />
+        <Grid className="kubevirt-cluster-overview__health-grid">
+          <GridItem span={12}>
+            <OCSHealthConnected />
+          </GridItem>
+          <GridItem span={12}>
+            <AlertsConnected className="kubevirt-cluster-overview__card--top-border" />
```
Comment (suomiy) [inline src/components/StorageOverview/StorageOverview.js]: can you please move this to `sass/Dashboard/dashboard.scss` if it is used in more places?

File: src/components/StorageOverview/StorageOverview.js [hunk 5]
```diff
@@ -12,12 +12,20 @@ import OCSHealthConnected from './OCSHealth/Health';
...
 import EventsConnected from './Events/Events';
 import { UtilizationConnected } from './Utilization/Utilization';
+import { AlertsConnected } from './Alerts/Alerts';
 
 const MainCards = () => (
   <GridItem lg={6} md={12} sm={12}>
     <Grid>
       <GridItem span={12}>
-        <OCSHealthConnected />
+        <Grid className="kubevirt-cluster-overview__health-grid">
```
Comment (suomiy) [inline src/components/StorageOverview/StorageOverview.js]: this also?

Comment (cloudbehl) [reply src/components/StorageOverview/StorageOverview.js]: ack

File: src/components/StorageOverview/StorageOverview.js (hunk 4 above)
Comment (cloudbehl) [reply src/components/StorageOverview/StorageOverview.js]: ack

**Comment by cloudbehl** [comment]
@suomiy made changes, please review.

**Comment by suomiy** [comment]
needs rebase

**Comment by cloudbehl** [comment]
@suomiy Ack, Done

**Pull request merged**